src/libraries/System.Speech/tests/SynthesizeRecognizeTests.cs

```diff
@@ -38,6 +38,7 @@ public void SpeechSynthesizerToSpeechRecognitionEngine()
             {
                 synth.SetOutputToWaveStream(ms);
                 var prompt = new Prompt("synthesizer");
+                synth.SetOutputToDefaultAudioDevice();
```

Comment:
Remove update to unit test as it doesn't work in CI due to our VM configurations

```suggestion
```